src/Data/HashTable/ST/Basic.hs

```diff
@@ -608,6 +665,10 @@ deletedMarker = 1
 
 
 ------------------------------------------------------------------------------
+trueInt :: Int -> Bool
```

Comment:
`tagToEnum#` is in `GHC.Prim`. Do you want me to import and use it directly, or should I use `isTrue#` from `GHC.Exts` for GHC >= 7.7 and leave the implementation with comparison for older versions of GHC?